integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -359,6 +364,36 @@ public static boolean isFuseInstalled() {
     return false;
   }
 
+  /**
+   * @return true if os version is matched to use macFuse
+   */
+  public static boolean isMacFuse() {
+    return compareVersion(SystemUtils.OS_VERSION, MACFUSE_SUPPORT_MINIMUM_OS_VERSION) >= 0;
+  }
+
+  /**
+   * Compares two version.
+   *
+   * @param version1 the first os version to compare
+   * @param version2 the second os version to compare
+   * @return the value 0 if version1 == version2; a value less than 0 if version1 < version2;
+   * and a value greater than 0 if version1 > version2
+   */
+  public static int compareVersion(String version1, String version2) {
```

Comment:
Are there existing java function or Alluxio java function do the same thing? so that we don't need to create our own

Reply:
I didn't find the relevant method in alluxio, ready to use ComparableVersion to replace the custom method.